### ISSUE TITLE
Mixer build bundles crashes in DNF call due to GPGME

### ISSUE THREAD

**Issue by tmarcu** [clearlinux/distribution#144]
### What is the issue type?

* [X] Bug report

### Describe the problem
Mixer build bundles crashes with the following error:

```
Traceback (most recent call last):
  File "/usr/bin/dnf", line 57, in <module>
    from dnf.cli import main
  File "/usr/lib/python3.7/site-packages/dnf/__init__.py", line 30, in <module>
    import dnf.base  
  File "/usr/lib/python3.7/site-packages/dnf/base.py", line 33, in <module>
    from dnf.module.repo_module_dict import RepoModuleDict
  File "/usr/lib/python3.7/site-packages/dnf/module/repo_module_dict.py", line 24, in <module>
    from dnf.conf.read import ModuleReader
  File "/usr/lib/python3.7/site-packages/dnf/conf/__init__.py", line 40, in <module>
    from dnf.conf.config import PRIO_DEFAULT, PRIO_MAINCONFIG, PRIO_AUTOMATICCONFIG
  File "/usr/lib/python3.7/site-packages/dnf/conf/config.py", line 23, in <module>
    from dnf.yum import misc
  File "/usr/lib/python3.7/site-packages/dnf/yum/misc.py", line 30, in <module>
    import dnf.crypto
  File "/usr/lib/python3.7/site-packages/dnf/crypto.py", line 35, in <module>
    from gpg import Context
  File "/usr/lib/python3.7/site-packages/gpg/__init__.py", line 101, in <module> 
    from . import core
  File "/usr/lib/python3.7/site-packages/gpg/core.py", line 34, in <module>
    from . import gpgme
  File "/usr/lib/python3.7/site-packages/gpg/gpgme.py", line 347, in <module> 
    GPGME_KEYLIST_MODE_LOCATE = _gpgme.GPGME_KEYLIST_MODE_LOCATE
AttributeError: module 'gpg._gpgme' has no attribute 'GPGME_KEYLIST_MODE_LOCATE
```

I can't reproduce this on my machine specifically, but @alexjch is experiencing it with the same build of Clear (24270) and both our machines are verify --fix clean.

**Comment by alexjch** [comment]
I fixed this in my dev machine with the following steps:

1- Check what bundles provide gpgme and which of those bundles are installed
```
sudo swupd search gpgme.py
```

2- Removed installed bundles that provide gpgme (```koji, package-utils, mail-utils-dev, and mixer```)

3- Re create the mix workspace
```
rm -rf /home/clr/mix
mkdir /home/clr/mix
cd mix
mixer init
sudo mixer build bundles --native```

working again.

**Comment by tmarcu** [comment]
I have mixer, package-utils, and mail-utils installed but I still don't get this error somehow.

**Comment by alexjch** [comment]
@phmccarty I have mixer bundle already installed probably since I installed clear in my system (a while ago). I cloned mixer-tools locally and set it up as directed by tmarcu and attempted to follow mixer tutorial from clearlinux.org, but I was unable to complete it since the command ```sudo mixer build bundles --native``` did not work. So no I did not uninstall mixer bundle. I had both the cloned code and the installed bundle and what I was trying to use was the mixer from the bundle.

Currently I still have both cloned code and mixer from bundle and I do not have reported issue after I removed and added bundles as described in previous comments.

I had multiple bundles that provided gpgme.py and since this is a wrapper to a library my sense is that maybe there was a different version of the library without the variable GPGME_KEYLIST_MODE_LOCATE that was causing the problem. I added back all the bundles I removed though and the problem did not came back.
